.github/workflows/helm_release.yaml

```diff
@@ -0,0 +1,134 @@
+name: K8ssandra Operator Helm Release
+
+on:
+  workflow_call:
+    inputs:
+      helm_repo:
+        description: 'Helm repository to use'
+        default: 'next'
+        required: false
+        type: string
+    secrets:
+      GCP_PROJECT_ID:
+        required: true
+      GCP_SA_KEY:
+        required: true
+      GCP_HELM_BUCKET_NAME:
+        required: true
+
+jobs:
+  helm-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Update PATH
+        run: |
+          echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
+
+      - name: Create env variables
+        run: ./.github/scripts/env_variables.sh
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: ${{ env.HELM_VERSION }}
+
+      - name: Install yq
```

Comment:
No need to install yq in github runners, it's already installed: https://github.com/actions/runner-images/blob/8361bb299ba9290b70b6ed8bf337456714c6516d/images/linux/scripts/installers/yq.sh#L4